env.d/grafana

```diff
@@ -8,6 +8,20 @@ GF_DATABASE_PASSWORD=pass
 # Auth
 GF_SECURITY_ADMIN_PASSWORD=pass
 
+# OAUTH
+GF_AUTH_GENERIC_OAUTH_ENABLED=true
+GF_AUTH_GENERIC_OAUTH_NAME=keycloak
+GF_AUTH_GENERIC_OAUTH_ALLOW_SIGN_UP=true
+GF_AUTH_GENERIC_OAUTH_CLIENT_ID=potsie
+GF_AUTH_GENERIC_OAUTH_CLIENT_SECRET=fa9e98ee-61a1-4092-8dac-1597da0c1bb0
+GF_AUTH_GENERIC_OAUTH_AUTH_URL=http://localhost:8080/auth/realms/fun-mooc/protocol/openid-connect/auth
+GF_AUTH_GENERIC_OAUTH_TOKEN_URL=http://keycloak:8080/auth/realms/fun-mooc/protocol/openid-connect/token
```

Comment:
```suggestion
GF_AUTH_GENERIC_OAUTH_AUTH_URL=http://keycloak:8080/auth/realms/fun-mooc/protocol/openid-connect/auth
GF_AUTH_GENERIC_OAUTH_TOKEN_URL=http://keycloak:8080/auth/realms/fun-mooc/protocol/openid-connect/token
```

Reply:
`keycloak` host name is used for host resolution in the docker network (server to server requests). The following URL is resolved by the web browser outside from this network, so this wont work.

Reply:
But in production, you are right: it should be the same host name.